### ISSUE TITLE
lint isort I001: forbid multiple imports on single line?

### ISSUE THREAD

**Issue by fschulze** [astral-sh/ruff#12239]
I tried to configure lint.isort to block multiple imports on single lines without enforcing the way multiple imports are split.

This should be reported as I001:
```python
from typing import Any, Callable, ContextManager
```

Both of these variants should not be reported:
```python
from typing import Any
from typing import Callable
from typing import ContextManager
```

```python
from typing import (
    Any,
    Callable,
    ContextManager,
)
```

The only setting which comes close is ``force-single-line=true``, but then only the first variant is allowed and the last variant is reported.

**Comment by MichaReiser** [comment]
This is interesting. Could you explain why you want to allow both variants?

The goal of `isort` is to have consistent import formatting. That's why it only allows exactly one variant.

**Comment by pmiddend** [comment]
I'm actually having problems even with the current state of things. I enabled `force-single-line` and then tried to format/check a source file with

```
from typing import Any, Callable, ContextManager
```

And it just found no problems with it. Am I doing something wrong here?
